addons/attributes/RscAttributes.hpp

```diff
@@ -1057,6 +1061,12 @@ class GVAR(RscAttributesGroup): GVAR(RscAttributesBase) {
         class ButtonOK: ButtonOK {};
         class ButtonCancel: ButtonCancel {};
     };
+    class buttons {
```

Comment:
```suggestion
    class Buttons {
```

I'm not certain how arma reacts to this, but it looks better this way.

Reply:
The config is not case-sensitive, I left it lowercase to match the other ones in display configs: `controls` and `controlsBackground`, but I agree it looks better capitalized 😄. Will change the others.